lib/update.js

```diff
@@ -25,6 +26,19 @@ function update (state, options) {
   })
 
   .then(function () {
+    return internals.request({
+      url: state.url + '/session',
+      method: 'PUT',
+      body: internals.serialise('session', options)
+    })
+  })
```

Comment:
I’m sorry I’ve missed a problem here. `account.update(options)` can be called with `options.password` only. That results in the `PUT /session` request to only send 

```json
{
    "data": {
        "type": "session",
        "attributes": {
            "password": "secretnew"
        }
    }
}
```

If `options.username` is not set, we must default it to `account.username`.

That also means that `account.update({username: 'newusername'}) will not work, we must require the password to be passed, too. 

The alternative would be that the server can set a new session ID in the server response, so that we wouldn’t need the `PATCH /session` request altogether. It would also be good because a password change does not require a session renewal as far as I know. 

Any thoughts?